flink-tests/src/test/java/org/apache/flink/test/misc/SuccessAfterNetworkBuffersFailureITCase.java

```diff
@@ -50,22 +50,22 @@
  */
 public class SuccessAfterNetworkBuffersFailureITCase extends TestLogger {
 
-    private static final int PARALLELISM = 16;
+    private static final int PARALLELISM = 4;
 
     @ClassRule
     public static final MiniClusterWithClientResource MINI_CLUSTER_RESOURCE =
             new MiniClusterWithClientResource(
                     new MiniClusterResourceConfiguration.Builder()
                             .setConfiguration(getConfiguration())
                             .setNumberTaskManagers(2)
-                            .setNumberSlotsPerTaskManager(8)
+                            .setNumberSlotsPerTaskManager(2)
                             .build());
 
     private static Configuration getConfiguration() {
         Configuration config = new Configuration();
-        config.set(TaskManagerOptions.MANAGED_MEMORY_SIZE, MemorySize.parse("80m"));
-        config.set(TaskManagerOptions.NETWORK_MEMORY_MIN, MemorySize.ofMebiBytes(32L));
-        config.set(TaskManagerOptions.NETWORK_MEMORY_MAX, MemorySize.ofMebiBytes(32L));
+        config.set(TaskManagerOptions.MANAGED_MEMORY_SIZE, MemorySize.parse("20m"));
+        config.set(TaskManagerOptions.NETWORK_MEMORY_MIN, MemorySize.ofMebiBytes(2L));
```

Comment:
should this be also just divided by four?

Reply:
Dividing by four would make the kmeans job pass. Hence, I needed to divide it by the 4^2.